### ISSUE TITLE
Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.redisson.spring.starter.RedissonAutoConfiguration

### ISSUE THREAD

**Issue by luamas** [redisson/redisson#4757]
2022-12-22 12:48:56 WARN  [main] org.springframework.core.LocalVariableTableParameterNameDiscoverer.inspectClass(LocalVariableTableParameterNameDiscoverer.java:123) : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.redisson.spring.starter.RedissonAutoConfiguration

**Expected behavior**

**Actual behavior**

**Steps to reproduce or test case**

**Redis version**

**Redisson version**
3.19.0
**Redisson configuration**

**Spring boot version**
3.0.0

**Comment by luamas** [comment]
maven-compiler-plugin
```
<groupId>org.apache.maven.plugins</groupId>
<artifactId>maven-compiler-plugin</artifactId>
<configuration>
	<compilerArgs>
		<arg>-parameters</arg>
	</compilerArgs>
</configuration>
```

**Comment by mrniko** [comment]
Fixed in Spring 6.0.3 https://github.com/spring-projects/spring-framework/issues/29612

**Comment by vishalsingh2972** [comment]
@mrniko @luamas Getting similar warning after trying everything....any resolution?

`Apr 14, 2023 1:09:54 AM org.springframework.core.LocalVariableTableParameterNameDiscoverer inspectClass
WARNING: Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: demo.LoggingAspect`

I am currently using Spring 6.0.7
